src/app/store/accounts/blockchain/stacks/stacks-account.hooks.ts

```diff
@@ -16,6 +16,14 @@ export function useStacksAccounts() {
   return useAtomValue(stacksAccountState);
 }
 
+export function useStacksAccount(accountIndex: number) {
+  const accounts = useStacksAccounts();
+  return useMemo(() => {
```

Comment:
I guess this `useMemo` could be helpful if users have tons of stacks accounts but I'm not sure if it's that much faster than just taking it from the state. 

I was checking into this and this is an area we can improve in the extension and eventually deprecate `jotai`. We have both `redux` + `jotai` state management there and this code `useStacksAccounts` calls `useAtomValue` on `stacksAccountState`. I guess it could be refactored to just use a redux selector?